tests/testem.json

```diff
@@ -0,0 +1,9 @@
+{
+    "test_page": "tests/all-tests.html",
+    "launch": "Chrome",
+    "browser_args": {
+        "Chrome": [
+            "--disable-extensions-except=dist/"
```

Comment:
I'm a little confused by this line.  From our discussion, sounds like you should just disable all extensions.